actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -1,9 +1,12 @@
 require 'thread_safe'
+require 'action_view/path_set'
 
 module ActionView
   class DependencyTracker # :nodoc:
     @trackers = ThreadSafe::Cache.new
 
+    mattr_accessor(:view_paths) { ActionView::PathSet.new }
```

Comment:
Right, I think you can override `view_paths`. Hm, not sure, how to catch those usages. Can we get a reference to the controller in those cases? If you happen to know that is.